src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -433,7 +433,7 @@ const ArtworkAppFragmentContainer = createFragmentContainer(
           extendedBiddingIntervalMinutes
         }
         saleMessage
-        artists {
+        artists(shallow: true) {
```

Comment:
I know this is already done on the MP side, and in general i'm always super opposed to 1:2 representations, but it'd be great to avoid the `shallow` `all` `short` abstractions on the MP side, and use something a little more literal

Reply:
What would you recommend? It seems like this should simply be documented on the `shallow` arguments within Metaphysics. Currently that looks like:

```
"Use whatever is in the original response instead of making a request"
```

Reply:
The comment says it all, though it'd be nice if we could summarize that comment in a way that [makes sense](https://www.merriam-webster.com/dictionary/shallow) for a variable name. GraphQL is our idealized UI for our backends. I don't know if any of us would chose this API if we could start again.

Reply:
Note that `shallow` _usually_ corresponds with the "short" JSON from the upstream API, but not always when `reference_properties` [are specified](https://github.com/mongoid/mongoid-cached-json?tab=readme-ov-file#reference-properties).

Reply:
It is what it is, and the above further emphasizes my point 🤷 Just a thought to consider GraphQL as UI, and noting a potential opportunity, but don't hold up anything for this comment 👍